worker/deps/openssl/openssl.gyp

```diff
@@ -21,12 +21,12 @@
         }, 'target_arch=="arm64" and OS=="win"', {
           # VC-WIN64-ARM inherits from VC-noCE-common that has no asms.
           'includes': ['./openssl_no_asm.gypi'],
-        }, 'gas_version and v(gas_version) >= v("2.26") or '
-           'nasm_version and v(nasm_version) >= v("2.11.8")', {
-           # Require AVX512IFMA supported. See
-           # https://www.openssl.org/docs/man1.1.1/man3/OPENSSL_ia32cap.html
-           # Currently crypto/poly1305/asm/poly1305-x86_64.pl requires AVX512IFMA.
-          'includes': ['./openssl_asm.gypi'],
+        #}, 'gas_version and v(gas_version) >= v("2.26") or '
```

Comment:
Why was this commented-out? It should (theoretically) use even faster ASM if I understand it correctly.

Reply:
Sorry for delay, I've seen now your comment. 
The block is commented because the `v()` macro is not available in node-gyp, and we have to assume that gas/nasm installed versions satisfy the requirement.